src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -656,7 +658,7 @@ namespace cryptonote
       return false;
     }
 
-    auto lns_db_file_path = folder / "lns.db";
+    auto ons_db_file_path = folder / "ons.db";
```

Comment:
I am a bit worried about this change: it means you will have to do a length blockchain rescan on startup because the ons.db file won't exist.

Maybe we could change this to check whether `ons.db` exists, and if so, use it.  Otherwise if `lns.db` exists, use that, and then if neither exist, use `ons.db`.

Reply:
will clean up in #1403